src/geometry/LinearPropagationHandler.hh

```diff
@@ -0,0 +1,47 @@
+//---------------------------------*-CUDA-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file LinearPropagationHandler.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "geometry/GeoTrackView.hh"
+#include "base/NumericLimits.hh"
+
+namespace celeritas
+{
+/*!
+ * \brief Handler applying linear propagation to neutral tracks.
+ */
+class LinearPropagationHandler
```

Comment:
I'd prefer to avoid class names with `Handler`, `Manager`, and other generic suffixes that may not contribute meaningfully to the understanding what a class does. Since this is a function-like class that just propagates the particle, can you rename to `LinearPropagator`?

Reply:
Class LinearPropagationHandler renamed as LinearPropagator, as suggsted.